xds/src/test/java/io/grpc/xds/XdsSdsClientServerTest.java

```diff
@@ -206,8 +213,14 @@ public void mtls_badClientCert_expectException() throws IOException, URISyntaxEx
           false);
       fail("exception expected");
     } catch (StatusRuntimeException sre) {
-      assertThat(sre).hasCauseThat().isInstanceOf(SSLHandshakeException.class);
-      assertThat(sre).hasCauseThat().hasMessageThat().contains("HANDSHAKE_FAILURE");
+      if (sre.getCause() instanceof SSLHandshakeException) {
+        assertThat(sre).hasCauseThat().isInstanceOf(SSLHandshakeException.class);
```

Comment:
Same as above. If `sre.getCause() instanceof SSLHandshakeException` is true, wouldn't this assertion will always be true?